aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -74,6 +74,12 @@ module aptos_framework::account {
         recipient_address: address,
     }
 
+    struct SignerCapabilityOfferProofChallengeV2 has drop {
```

Comment:
May be we can also throw expiration in here as well to be safe?

Reply:
What do you mean by `expiration`? The sequence number is effectively an expiration…